pulp_rpm/app/models.py

```diff
@@ -784,3 +791,254 @@ class ModulemdDefaults(Content):
+    identity = <str>                      ; path to a pem file that identifies a product
+    [addon-$addon_uid]
+    id = <str>                            ; addon ID
+    uid = <str>                           ; addon UID ($parent_UID.$ID)
+    name = <str>                          ; addon name
+    type = addon
+    ; addon paths
+    ; see variant paths
+    [general]
+    ; WARNING.0 = This section provides compatibility with pre-productmd treeinfos.
+    ; WARNING.1 = Read productmd documentation for details about new format.
+    family = <str>                        ; equal to [release]/name
+    version = <str>                       ; equal to [release]/version
+    name = <str>                          ; equal to "$family $version"
+    arch = <str>                          ; equal to [tree]/arch
+    platforms = <str>[,<str>...]          ; equal to [tree]/platforms
+    packagedir = <str>                    ; equal to [variant-*]/packages
+    repository = <str>                    ; equal to [variant-*]/repository
+    timestamp = <int>                     ; equal to [tree]/build_timestamp
+    variant = <str>                       ; variant UID of first variant (sorted alphabetically)
+    ```
+    Older "treeinfo" file formats exist. They vary from repository to repository, but look
+    similar to the following:
+    ```
+    [general]
+    name = Fedora-Server-21
+    family = Fedora-Server
+    timestamp = 1417653911.68
+    variant = Server
+    version = 21
+    packagedir =
+    arch = x86_64
+    [stage2]
+    mainimage = LiveOS/squashfs.img
+    [images-x86_64]
+    kernel = images/pxeboot/vmlinuz
+    initrd = images/pxeboot/initrd.img
+    upgrade = images/pxeboot/upgrade.img
+    boot.iso = images/boot.iso
+    [images-xen]
+    kernel = images/pxeboot/vmlinuz
+    initrd = images/pxeboot/initrd.img
+    upgrade = images/pxeboot/upgrade.img
+    [checksums]
+    images/efiboot.img = sha256:de48c8b25f03861c00c355ccf78108159f1f2aa63d0d63f92815146c24f60164
+    images/macboot.img = sha256:da76ff5490b4ae7e123f19b8f4b36efd6b7c435073551978d50c5181852a87f5
+    images/product.img = sha256:ffce14a7a95be20b36f302cb0698be8c19fda798807d3d63a491d6f7c1b23b5b
+    images/boot.iso = sha256:56af126a50c227d779a200b414f68ea7bcf58e21c8035500cd21ba164f85b9b4
+    images/pxeboot/vmlinuz = sha256:81c28a439f1d23786057d3b57db66e00b2b1a39b64d54de1a90cf2617e53...
+    images/pxeboot/initrd.img = sha256:aadebd07c4c0f19304f0df7535a8f4218e5141602f95adec08ad1e22f...
+    images/pxeboot/upgrade.img = sha256:224d098fb3903583b491692c5e0e1d20ea840d51f4da671ced97d422...
+    repodata/repomd.xml = sha256:3af1609aa27949bf1e02e9204a7d4da7efee470063dadbc3ea0be3ef7f1f4d14
+    ```
+    Note that not all fields may be present. For example, some RHEL 5 repositories do not
+    contain the `variant` field in their "treeinfo" files.
+    :ivar distribution_id: The concatenation of the values of `family`, `variant`, `version`,
```

Comment:
This field descriptions are from pulp 2 and they are not relevant here.
Please, add the description of the fields for this class.